generator/config/stage/collStats.yaml

```diff
@@ -11,3 +11,33 @@ arguments:
         name: config
         type:
             - object
+tests:
+    -
+        name: 'latencyStats Document'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/collStats/#latencystats-document'
+        pipeline:
+            -
+                $collStats:
+                    latencyStats:
+                        histograms: true
```

Comment:
The config contains nested object definition. That's why I kept a generic "config" object. DX could be improved with advanced type in phpdoc.

Reply:
SGTM. I don't think this stage will be used very often.

Reply:
Ok, I added the 4 parameters.